pkg/providers/tinkerbell/stack/testdata/expected_with_docker_options.yaml

```diff
@@ -34,8 +34,7 @@ rufio:
 tinkController:
   image: public.ecr.aws/eks-anywhere/tink-controller:latest
 tinkServer:
-  args:
-  - --tls=false
```

Comment:
I noticed that TLS is still set to false in get boots env. Was this never necessary or is this related to something else that change for this to be removed?

Reply:
It wasn't necessary.